tests-hal-configuration/src/test/java/org/jboss/hal/testsuite/test/configuration/SubsystemsTest.java

```diff
@@ -56,8 +57,8 @@
     private static final Operations operations = new Operations(client);
     private static List<String> subsystems = new ArrayList<>();
     // this is from org.jboss.hal.client.configuration.subsystem.SubsystemColumn
-    private static final List<String> EMPTY_SUBSYSTEMS = asList("bean-validation", "ee-security", "jaxrs", "jdr",
```

Comment:
The `jaxrs` was no longer empty?

Reply:
It's look like it. When I try EAP 7.3.9 it's look empty but 7.4.x have content in Data and Attributes and in hal/console repo jaxrs was enabled by HAL-1633. So from this two things i assume it's usable and it's not empty. 
When I think about this the test should be written for this subsystem as well.